deepspeed/runtime/engine.py

```diff
@@ -1406,10 +1410,8 @@ def _configure_zero_optimizer(self, optimizer):
 
         elif zero_stage == ZeroStageEnum.weights:
             assert not self.has_moe_layers, "MoE not supported with Stage 3"
-            logger.info("Initializing ZeRO Stage 3") if dist.get_rank() == 0 else None
-            from deepspeed.runtime.zero.stage3 import DeepSpeedZeroOptimizer_Stage3
-
             if isinstance(optimizer, DummyOptim):
+                logger.info("Creating ZeRO Offload") if dist.get_rank() == 0 else None
```

Comment:
Can we replace this with `log_dist` (and maybe the logger call on [line 302](https://github.com/microsoft/DeepSpeed/blob/49c1f78ddcbb794acb2f1fbb4c3d1866ff0d431c/deepspeed/runtime/engine.py#L302) as well)

Reply:
Done